Makefile

```diff
@@ -114,6 +114,16 @@ lib-release-test: lib-package
 lib-release: lib-package
 	twine upload dist/*
 
+lib-doc:
+	pushd docs && \
+	python -m sphinx \
+		--keep-going \
+		-b dirhtml \
+		-d _build/doctrees \
+		-D language=en . _build/html && \
+	popd && \
+	open docs/_build/html/index.html
```

Comment:
Also, `open` only exists on macOS. Here you exclude all the Linux-based contributors who'd need an `xdg-open.

Reply:
Thank you, will consider presence of `xdg-open` or add a OS check